README.md

```diff
@@ -2,15 +2,36 @@
 
 UX Aspects Seed Project provides a skeleton which can be used to build an application using [UX Aspects](https://uxaspects.github.io/UXAspects).
 
-## Getting Started
-
-##### Install with Bower
-* Install [Git](https://git-scm.com/downloads)
-* run `git clone https://github.com/UXAspects/UXAspects-Seed`
-* Install [Nodejs](https://nodejs.org/en/)
-  * Note: Node comes with [npm](https://docs.npmjs.com/getting-started/installing-node) installed but npm is updated more frequently than Node
-* Run  `npm install` from the folder where the code will be downloaded
-* Install [bower(>v1.5)](http://bower.io/#install-bower) using npm `npm install -g bower`
-* Run `bower install` in the seed project folder
-* Run `npm start`
-* Browse "http://localhost:8080/"
+# Installation
```

Comment:
Can you call this "Getting Started" or similar? This isn't really an installation.